src/components/FilePreview/FilePreview.tsx

```diff
@@ -0,0 +1,129 @@
+import React, {useEffect, useId, useState} from 'react';
```

Comment:
useId usage is breaking change, this library is compatible up to react 16, as peerDependencies